### PR TITLE
refactor(frontend): style and design changes to a contribution

### LINKED ISSUE #2328: 🚀 [Feature] Show History entries in the Contributions Conversation
## 🚀 Feature

Show History entries in the Contributions Conversation.

### LINKED ISSUE #2384: 🔧 [Refactor] Mark History Contribution Message properly
## 🔧 Refactor ticket

Mark History Contribution Message properly

- [ ] Graphical distinction for contribution Message type `HISTORY`
- [ ] Consider also a text/description for this type

### LINKED ISSUE #2562: 🔧 [Refactor] style and design changes to a contribution
## 🔧 Refactor ticket
These places need to be revised.

- [ ]  displaying changes to a contribution in moderator chat
image

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
Display of an edited contribution improved

### Issues
- fixes #2562 
- fixes #2328 
- fixes #2384 

Bildschirmfoto vom 2023-02-01 13-54-03

### CODE REVIEW

**Comment by ulfgebhardt** [comment]
Great improvement!

I would like to have the entry changes with a different background color. I suggest a faint green.

File: frontend/src/locales/de.json [hunk 1]
```diff
@@ -67,6 +68,7 @@
       "lastMonth": "Für den ausgewählten Monat ist das Schöpfungslimit erreicht.",
       "thisMonth": "Für den aktuellen Monat ist das Schöpfungslimit erreicht."
     },
+    "oldContribution": "Alter Beitrag",
```
Comment (Mogge) [inline frontend/src/locales/de.json:71]:
```suggestion
    "oldContribution": "Vorherige Version",
```

File: frontend/src/locales/en.json [hunk 2]
```diff
@@ -67,6 +68,7 @@
       "lastMonth": "The creation limit is reached for the selected month.",
       "thisMonth": "The creation limit has been reached for the current month."
     },
+    "oldContribution": "Old post",
```
Comment (Mogge) [inline frontend/src/locales/en.json:71]:
```suggestion
    "oldContribution": "Previous version",
```

**Review by Mogge — CHANGES_REQUESTED** [review]

File: frontend/src/locales/de.json [hunk 3]
```diff
@@ -56,6 +56,7 @@
       "openAmountForMonth": "Für <b>{monthAndYear}</b> kannst du noch <b>{creation}</b> GDD einreichen.",
       "yourContribution": "Dein Beitrag zum Gemeinwohl"
     },
+    "isEdited": "hat seinen Beitrag bearbeitet",
```
Comment (Mogge) [inline frontend/src/locales/de.json:59]:
```suggestion
    "isEdited": "hat den Beitrag bearbeitet",
```

File: frontend/src/locales/en.json [hunk 4]
```diff
@@ -56,6 +56,7 @@
       "openAmountForMonth": "For  <b>{monthAndYear}</b>, you can still submit <b>{creation}</b> GDD.",
       "yourContribution": "Your Contributions to the Common Good"
     },
+    "isEdited": "edited his contribution",
```
Comment (Mogge) [inline frontend/src/locales/en.json:59]:
```suggestion
    "isEdited": "edited the contribution",
```

**Review by Mogge — CHANGES_REQUESTED** [review]
I do not know why I did not see this on the first glance ...

**Pull request merged**